### PR TITLE
Changes Related to wETH deposit and withdrawals backfill substream

### PR DESCRIPTION
**Pull request by Hrishikesh-Thakkar**
Screenshot 2024-02-09 at 7 15 43 PM
Screenshot 2024-02-09 at 7 16 11 PM

### CODE REVIEW

File: transfer-tracker/src/lib.rs [hunk 1]
```diff
@@ -151,6 +151,78 @@ fn map_transfers(blk: eth::Block) -> Result<Transfers, Error> {
             };
             erc1155_transfer_singles.push(erc1155_transfer_single);
         }
+        
+        let token_address = helpers::utils::format_with_0x(
+            Hex(log.clone().address()).to_string(),
+        );
+        if token_address == "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2".to_string(){
```
Comment (0xsarvesh) [inline transfer-tracker/src/lib.rs]: You can move this to yaml file.

File: transfer-tracker/src/lib.rs [hunk 2]
```diff
@@ -151,6 +151,78 @@ fn map_transfers(blk: eth::Block) -> Result<Transfers, Error> {
...
+                        Hex(log.clone().address()).to_string(),
+                    ),
+                    chain_id: 1.to_string(),
+                    log_index: log.block_index(),
+                    source: 1,
+                    transaction_hash: helpers::utils::format_with_0x(tx_hash.clone()),
+                    operator: helpers::utils::format_with_0x(
+                        Hex(log.receipt.transaction.from.clone()).to_string(),
+                    ),
+                    from: String::from("0x0000000000000000000000000000000000000000"),
```
Comment (0xsarvesh) [inline transfer-tracker/src/lib.rs]: Take out zero address as constant

File: transfer-tracker/src/lib.rs (hunk 1 above)
Comment (0xsarvesh) [reply transfer-tracker/src/lib.rs]: This address is chain specific?

**Review by 0xsarvesh — CHANGES_REQUESTED** [review]
Overall looks good, some minor feedbacks

File: transfer-tracker/src/lib.rs (hunk 1 above)
Comment (Hrishikesh-Thakkar) [reply transfer-tracker/src/lib.rs]: Yes only for ethereum

**Review by 0xsarvesh — APPROVED** [review]

**Pull request merged**